cmd/thanos/flags.go

```diff
@@ -124,3 +125,46 @@ func modelDuration(flags *kingpin.FlagClause) *model.Duration {
 
 	return value
 }
+
+type pathOrContent struct {
```

Comment:
would be tempted to call this `storeConfig`

Reply:
Not really - it's not tied to store in any way.